### DISCUSSION TITLE
`undergraduate-thesis`及`paper-translation`与2023年模板的区别

### DISCUSSION THREAD

**Discussion by YDX-2147483647** [BITNP/BIThesis#396] [General]
> [!NOTE]
>
> 这里只比`undergraduate-thesis`的外观，下面评论比文字要求和`paper-translation`。

对比的文件：

- `main.pdf`

  使用 TeX Live 2023 (Windows) 编译 https://github.com/BITNP/BIThesis/tree/0d2315f1f91f26bd9ecf686040f46085f48f9672/templates/undergraduate-thesis 。

- `北京理工大学本科生毕业设计（论文）模板（2023年12月）.pdf`

  用 MS Word 导出`北京理工大学本科生毕业设计（论文）模板（2023年12月）.docx`为 PDF。

对比方法：

- [Diffpdf Windows version | RubyPdf Technologies](https://soft.rubypdf.com/software/diffpdf) 2.1.3
- 只比外观

# 外观

## 主要区别：间距

每页第一个标题上方、表格标题下方等处的间距不太一样。

![图片](https://github.com/BITNP/BIThesis/assets/73375426/40185097-ef88-469e-8d34-0b51b7b3f62a)
![屏幕截图 2023-12-31 004408](https://github.com/BITNP/BIThesis/assets/73375426/f22a5863-53a8-48d7-af4f-583d7f900170)
![屏幕截图 2023-12-31 004111](https://github.com/BITNP/BIThesis/assets/73375426/eeb232dc-646c-4f3c-b0fe-f3926b4a0e44)

### 标点避头尾策略

![屏幕截图 2023-12-31 003709](https://github.com/BITNP/BIThesis/assets/73375426/77281336-7766-499f-aedc-3b325e840b73)

### 目录中，英文字体、罗马数字形式

Word 模板：英文使用了宋体附带的英文部分，罗马数字没有使用 Unicode 中专门的字符。

![屏幕截图 2023-12-31 004008](https://github.com/BITNP/BIThesis/assets/73375426/b5634d7b-0850-4db2-82a3-a0eb31e28a7b)

## 其它：示例

- 公式中正体、斜体

  Word 模板：使用已过时的 Equation Editor 3.0，全都是正体。

  ![屏幕截图 2023-12-31 004140](https://github.com/BITNP/BIThesis/assets/73375426/72da1c0a-83cf-4523-bf29-7e4ad415b11c)

- 表格“合计”上方横线

  （参考开头第二张图）

## 个人观点

- 间距有时间就改，没时间不改也行。
- 次要区别几乎是 Word 模板的“错误”，不应该改。

**Comment by YDX-2147483647** [comment]
涉及的官方文件：`北京理工大学本科生毕业设计（论文）书写规范及打印装订要求（2023年12月）.doc`，该文件是`北京理工大学本科生毕业设计（论文）指导手册（2023年12月）.docx`的结尾（文字 identical）。

# 文字要求

## 文档格式要求·页面设置

- A4纸纵向打印；

- 页边距：上3.5cm、下2.6cm、左3cm、右2.6cm；页眉：2.4cm、页脚：2cm、装订线：0cm；

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L940-L951

- 页眉内容为：“北京理工大学本科生毕业设计（论文）”页眉，宋体、四号，居中排列，字间距加宽0.5磅； 页脚内容为页码，宋体、五号，居中排列。

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1253-L1259

## 文档格式要求·文档格式／标题序号标号

- 一级标题：黑体，三号，加粗；间距：段前0.5行，段后1行；居中；

  > 第1章     第2章    第3章       ……

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1267-L1280

- 二级标题：黑体，四号，加粗；间距：段前0.5行，段后0行；居左对齐；

  > 1.1   1.2    1.3      ……

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1282-L1292

- 三级标题：黑体、小四、加粗；间距：段前0.5行，段后0行；居左对齐；

  > 1.1.1  1.1.2  1.1.3    ……

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1294-L1310

- 标题行距：1.5倍行距；

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1463-L1470

- 目录：一级标题居左对齐，下一级标题依次向右缩进一格。

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1352-L1363

- 正文部分：宋体、小四；正文行距：22磅；间距段前段后均为0行。

  https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1425-L1427

## 图、表标号

采用阿拉伯数字按章依序编码。

> 图1-1    图1-2 ……    图2-1    图2-2    ……
>
> 表1-1    表1-2 ……    表2-1    表2-2    ……

图、表居中，图注标在图下方，表头标在表上方，宋体、五号、居中，图表与上下文之间各空一行。

https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1490-L1496

## 公式标号

采用阿拉伯数字按章依序编码。

> （1-1） （1-2）…… （2-2） （2-2）……

公式标注应于该公式所在行的最右侧。对于较长的公式只可在符号处（+、-、*、/、≤≥等）转行。在文中引用公式时，在标号前加“式”，如式（1-2）。

https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L1498-L1499

## 参考文献书写规范

参考国标【GB/T 7714—2015】，参考文献书写规范如下：……

（略）

关于参考文献的未尽事项可参考国家标准《信息与文献参考文献著录规则》（GB/T 7714—2015）

## 附录

附录是论文主体的补充项目，……（略）

- 附录依次用大写正体英文字母A、B、C……编序号，如附录A，附录B。 
- 附录中的图、表、公式、参考文献等另行用阿拉伯数字编序号，与正文分开，也一律用阿拉伯数字编码，但在数码前冠以附录序码，如：图A-1；表B-2；式（B-3）；文献[A-4]等。

https://github.com/BITNP/BIThesis/blob/0d2315f1f91f26bd9ecf686040f46085f48f9672/bithesis.dtx#L2739-L2772

**Comment by YDX-2147483647** [comment]
# `paper-translation`

对比的文件：

- `main.pdf`

  同法编译 https://github.com/BITNP/BIThesis/tree/0d2315f1f91f26bd9ecf686040f46085f48f9672/templates/paper-translation 。

- `外文翻译-封面.pdf`

  同法用`外文翻译-封面.doc`生成。

`北京理工大学本科生毕业设计（论文）书写规范及打印装订要求（2023年12月）.doc`要求如下，较为宽松。

> 外文翻译5000字左右，内容与毕业设计（论文）题目有关。外文原文用A4纸打印或复印，中文翻译用A4纸打印，文档格式要求参考“毕业设计（论文）文档格式要求”。装订顺序为：外文翻译封面→外文原文→中文翻译。


## 外观

- 间距不同。
- `{外文原文,中文翻译}题目：`字体不同。

![图片](https://github.com/BITNP/BIThesis/assets/73375426/a37d994d-59b8-40d5-8b7f-33cd32896227)


## 2024年3月5日补充

刚才信息学院教学干事通知说页眉是“北京理工大学本科生毕业设计（论文）”，不能改成“……外文翻译”。如果需要更改，请编辑`main.tex`。

```diff
  style = {
-   % head = {自定义页眉文字}
+   head = {北京理工大学本科生毕业设计（论文）}
  }
```

https://github.com/BITNP/BIThesis/issues/431

**Reply by fky2015** [reply]
word 模板如果把那个注释删除掉呢？

**Reply by YDX-2147483647** [reply]
没有区别，注那个文本框的环绕方式是“浮于文字上方”。

（不过已经把 #399 合并了啊）

**Comment by fky2015** [comment]
> 没有区别，注那个文本框的环绕方式是“浮于文字上方”。
> 
> （不过已经把 #399 合并了啊）

是的，合并了之后才想起来要问一下。

**Comment by YDX-2147483647** [comment]
## 2024年5月15日更新

大家在学习委员群里提出：

- 图表题头的数字：Word模板是宋体，但正文中数字是 Times；《书写规范》给的例子和LaTeX模板都是 Times。
- 标题中出现的数字：Word模板、《书写规范》是黑体，LaTeX模板是 Times New Roman。
- 此外，Word模板比较奇怪，目录中英文“Abstract”是宋体，“第1章”的“1”是Tahoma。LaTeX模板比较一致。

教学干事问了教务部。
